lib/apress/selenium_products/spec/company_site/product_creation_page/new_form_spec.rb

```diff
@@ -0,0 +1,621 @@
+    end
+  end
+
+  # Одинаковые поля в БЛ и ПЦ заполняют батарейку на разные проценты
+  describe 'Когда проверяется батарейка' do
+    before { navigate_to_product_creation_page }
+
+    context 'когда выполняется клик по "Все параметры" в плашке с батарейкой' do
+      before { @product_creation_page.thermometer_degree_popup_link }
+
+      it 'откроется попап со справочной информацией о рассчете батарейки' do
+        expect(@product_creation_page.thermometer_degree_popup?).to be_truthy
+      end
+    end
+
+    context 'когда заполнена Цена' do
+      let(:options) { {price: CONFIG['product_creation']['price']['exact']} }
+
+      before { @product_creation_page.fill_price(options) }
+
+      it "значение батарейки #{CONFIG['battery_percents']['price']}%" do
+        expect(@product_creation_page.thermometer_degree).to eq "#{CONFIG['battery_percents']['price']}%"
+      end
+    end
+
+    context 'когда загружено Изображение' do
+      let(:options) { {path_to_image: IMAGE_PATH} }
+
+      before { @product_creation_page.load_image(options) }
+
+      it "значение батарейки #{CONFIG['battery_percents']['image']}%" do
+        expect(@product_creation_page.thermometer_degree).to eq "#{CONFIG['battery_percents']['image']}%"
+      end
+    end
+
+    context 'когда заполнено Краткое описание' do
+      let(:options) { { short_description: CONFIG['product_creation']['short_description']['valid'] } }
+
+      before { @product_creation_page.short_description = options[:short_description] }
+
+      it "значение батарейки #{CONFIG['battery_percents']['short_description']}%" do
+        expect(@product_creation_page.thermometer_degree).to eq "#{CONFIG['battery_percents']['short_description']}%"
+      end
+    end
+
+    context 'когда заполнено Полное описание' do
+      let(:options) { { description: CONFIG['product_creation']['full_description'] } }
+
+      before { @product_creation_page.description = options[:description] }
+
+      it "значение батарейки #{CONFIG['battery_percents']['description']}%" do
+        expect(@product_creation_page.thermometer_degree).to eq "#{CONFIG['battery_percents']['description']}%"
+      end
+    end
+
+    context 'когда заполнено Наличие' do
+      let(:options) { { exists: CONFIG['product_creation']['exists']['in_stock'] } }
+
+      before { @product_creation_page.exists_select = options[:exists] }
+
+      it "значение батарейки #{CONFIG['battery_percents']['exists']}%" do
+        expect(@product_creation_page.thermometer_degree).to eq "#{CONFIG['battery_percents']['exists']}%"
+      end
+    end
+
+    context 'когда заполнены все основные поля' do
+      let(:options) do {
+        price: CONFIG['product_creation']['price']['exact'],
+        path_to_image: IMAGE_PATH,
+        short_description: CONFIG['product_creation']['short_description']['valid'],
+        description: CONFIG['product_creation']['full_description'],
+        exists: CONFIG['product_creation']['exists']['in_stock'] }
```

Comment:
[Correctable] Layout/SpaceInsideHashLiteralBraces: Space inside } detected.<br>[Correctable] Layout/MultilineHashBraceLayout: Closing hash brace must be on the line after the last hash element when opening brace is on a separate line from the first hash element.